Expensify/App
# Fix up workspace routes

cc @parasharrajat we didn't really implement this 100% correctly as the routes need to be hooked up to the policy

### Details
- Hooks the workspace sidebar up to the chosen policy
- Enables tapping on that policy in `/settings`

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/168732

### Tests
1. Create a new workspace
![2021-06-25_12-47-21](https://user-images.githubusercontent.com/32969087/123491878-94f6da00-d5b3-11eb-84dd-efb2e566e86b.png)
![2021-06-25_12-47-34](https://user-images.githubusercontent.com/32969087/123491880-97593400-d5b3-11eb-9c9c-6976272ab07c.png)
![2021-06-25_12-47-40](https://user-images.githubusercontent.com/32969087/123491884-99bb8e00-d5b3-11eb-9e52-d843c3cfd4d9.png)
2. Verify you are redirected to the `workspace/:policyID/card` route + the sidebar has the workspace name
3. Close the modal
4. Navigate to `/settings` by clicking on your avatar
5. Verify the workspace appears in `/settings`/
6. Tap on the workspace name
![2021-06-25_12-48-32](https://user-images.githubusercontent.com/32969087/123491918-b8218980-d5b3-11eb-93cf-dbbd7d68011b.png)
7. Verify you are brought back to the `workspace/:policyID/card` route

### QA Steps


### Tested On


### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android
